content/pages/why.md

```diff
@@ -0,0 +1,89 @@
+Title: Why SaltyRTC?
+Date: 2016-12-03 19:00
+Modified: 2016-12-03 19:00
+Sort-Order: 1
+Summary: What is SaltyRTC all about and why should I use it?
+
+# Short
+
+Here's why you should use SaltyRTC. With SaltyRTC,
+
+* signalling data is fully end-to-end encrypted,
+* zero trust towards the server is needed,
+* no data is stored on the server,
+* servers can be self-hosted,
+* it doesn't fall apart without SSL/TLS, and
+* it's not limited to WebRTC and ORTC.
+
+# Long
+
+We'll take WebRTC signalling solutions as an example to describe signalling 
+types and their security issues.
+
+Most existing applications that utilise WebRTC claim that their users are fully 
+end-to-end encrypted because WebRTC provides end-to-end encryption. However, 
+this is not true. Because if the signalling is not end-to-end encrypted, any 
+two peers may connect to one another but it is not guaranteed that one peer 
+connects to another peer it actually wanted to establish the connection with. A 
```

Comment:
Maybe "If the signalling is not end-to-end encrypted, it's not guaranteed that the peer you connected to is actually the peer you originally wanted to establish the connection with."